lib/access_lint/rails/engine.rb

```diff
@@ -5,7 +5,7 @@ class Engine < ::Rails::Engine
 
       initializer "access_lint-rails" do |app|
         app.config.assets.precompile << "accesslint.js"
-
+        app.config.filter_parameters << :accesslint
```

Comment:
Is it possible in the engine to check for the environment, or to declare this filter parameter in `development.rb`?

I don't want to burn too much of your time here, I could just see debugging a problem on production being very confusing if this data was missing.